src/main/java/com/auth0/utils/Asserts.java

```diff
@@ -4,24 +4,61 @@
 import okhttp3.HttpUrl;
 
 public abstract class Asserts {
```

Comment:
Added JavaDocs and removed runtime exceptions from method signatures per best practices.